app/app.go

```diff
@@ -259,3 +271,92 @@ func (a *App) HandleBlock(blk *blockchain.Block) {
 		log.Debug("added blk number %d to chain", blk.BlockNumber)
 	}
 }
+
+// SyncBlockchain updates the local copy of the blockchain by requesting missing
+// blocks from peers. Returns error if we are not connected to any peers.
+func (a *App) SyncBlockchain() error {
```

Comment:
Nitpick, lets use `SyncBlockChain`? (since the blockchain itself is pascal case)